package.json

```diff
@@ -121,5 +118,8 @@
     "webpack": "^1.12.14",
     "webpack-dev-server": "^1.14.1",
     "webpack-merge": "^0.9.0"
+  },
+  "dependencies": {
```

Comment:
!fix Please move it to devDependencies

PS: Why? Because we use npm publish approach with built dist directory, so we don't need packages in dependecy block, because these packages will be installed when we install ```smart-house-frontend``` package